man/man8/zfs.8

```diff
@@ -1275,6 +1275,19 @@ compression algorithm compresses runs of zeros.
 This property can also be referred to by its shortened column name
 .Sy compress .
 Changing this property affects only newly-written data.
+.Pp
+When any setting except
+.Sy off
+is selected, compression will explicitly check for blocks consisting of only
+zeroes (the NUL byte).  When a zero-filled block is detected, it is stored as
+a hole and not compressed using the indicated compression algorithm.
+.Pp
+Any block being compressed must be no larger than 7/8 of its original size,
+otherwise the compression will be considered a failure and the block saved
+uncompressed. Note that when the filesystem block is less than 8 times the disk
+sector size this effectively reduces the necessary compression ratio; for
+example 8k blocks on disks with 4k disk sectors must compress to 1/2 or less of
```

Comment:
Might be personal preference but I'd suggest writing out "half" instead of 1/2.

Reply:
I still prefer `1/2` for consistency with the `7/8` above it and generally keeping the theme of real numbers to go with the `4k and 8k` sector/block sizes.

Reply:
Sure, OK.